python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -172,6 +175,37 @@ def resolve_partitionKeys(self, _graphene_info):
                 ]
         return []
 
+    def resolve_latestMaterializationPerPartition(self, _graphene_info, **kwargs):
+        from ..implementation.fetch_assets import get_asset_events
+
+        get_timestamp = lambda event: event.timestamp
+        get_partition = (
+            lambda event: event.dagster_event.step_materialization_data.materialization.partition
+        )
+
+        partitions = kwargs.get("partitions")
+        events_for_partitions = get_asset_events(
```

Comment:
I suspect that we will want to create storage-specific methods for these in the near future, but not to block this diff or any UI work.